docs/tutorial.rst

```diff
@@ -1264,6 +1264,16 @@ or by the index in ``$arg<n>`` environment variables.
     >>> aliases['piu'] = 'pip install -U @($args)'
     >>> aliases['cdls'] = 'cd $arg0 && ls'
 
+.. warning:: That means, if you need `@()` in your alias, you can't rely on
```

Comment:
```suggestion
.. warning:: That means, if you need ``@()`` in your alias, you can't rely on
```

Reply:
Sorry, yes. I'm too stuck in my Markdown ways.

Reply:
I miss the double backtick every time -- one of these days I'll port all the docs over to markdown but that's on my infinitely long todo list :shrug: